p2panda-js/README.md

```diff
@@ -51,9 +51,34 @@ To install `p2panda-js` from the NPM package, simply run:
 
 `npm i p2panda-js`
 
+## Usage
+
+Create a key pair for each device and user who need to access p2panda.
+
+```
+import { KeyPair } from 'p2panda-js';
+const keyPair = new KeyPair();
+```
+
+Create an instance using an already known schema for chat messages.
+
+```
+import { Session } from 'p2panda-js';
+
+const session = new Session('https://welle.liebechaos.org')
+  .keyPair(keyPair);
+
+const payload = {
+  message: 'Hi there'
+}
+const entry = await session.create(payload, { schema: CHAT_SCHEMA })
+
+```
```

Comment:
Some smol nitpicks here since examples are important:

* Add syntax highlighting
* Consistent JavaScript style (semi or not?), empty line at the end
* Make this example actually work, so one can just copy/paste it (define `CHAT_SCHEMA`, move all imports together to the top, use JS comments instead of multiple markdown code blocks with text inbetween)
* Could be nice to write a little bit more what every line is doing (aka "is session.create sending a message"?)
* *instance* is a little mysterious for a young pandawan, maybe we can write "Send a new chat message to the server" or similar?

Reply:
Great comments! We could also add something about querying entries / instances and the current limitations there.

Reply:
Should we use "node" instead of "server" here? eg "Open a long running connection to a p2panda node", "Send a new chat message to the node"

Reply:
I updated the example with the recommendations. I also used "node" to describe what we were calling "server".

Reply:
Actually, I just notice that `KeyPair`, which is used in the example, is only available by directly accessing it from the `wasm` export. But one goal of this pr was to have an API that doesn't require awaiting the import (`const { KeyPair } = await wasm;`) when you want to use `p2panda-js`.

So I added functions to create and restore key pairs directly.